dns/ddclient/src/opnsense/mvc/app/models/OPNsense/DynDNS/DynDNS.xml

```diff
@@ -69,6 +69,7 @@
                         <inwx>INWX</inwx>
                         <keysystems>Key-Systems</keysystems>
                         <loopia>Loopia</loopia>
+                        <mythicdyn>Mythic Beasts API V2</mythicdyn>
```

Comment:
If we only ever have one available the "API V2" annotation doesn't hold much value. People still need to check documentation on these and see which software supports what (being a ddclient frontend only).
```suggestion
                        <mythicdyn>Mythic Beasts</mythicdyn>
```